.github/workflows/reference_linter.yaml

```diff
@@ -0,0 +1,32 @@
+name: Lint Reference Files
```

Comment:
```suggestion
name: Lint Fluent Files
```

Reply:
Made all the other changes, used `Lint Fluent Reference Files` here, since we're not linting all the files (only en-US, i.e. reference)